include/sdsl/coder_elias_delta.hpp

```diff
@@ -191,7 +194,8 @@ bool elias_delta::encode(const int_vector& v, int_vector& z)
 	return true;
 }
 
-inline void elias_delta::encode(uint64_t x, uint64_t*& z, uint8_t& offset)
+template <typename T>
```

Comment:
I have been thinking about an alternative to ``T`` that both matches snake_case and is a bit more meaningful but I didn't come up with one yet :woman_shrugging: